File: p-1/solution.go
package main

import "fmt"

func multiples(n int) int {
	sum := 0

	for i := n - 1; i > 0; i-- {
		if i % 3 == 0 || i % 5 == 0 {
			sum += i
		}
	}

	return sum
}

func main() {
	fmt.Println(multiples(10))
	fmt.Println(multiples(100))
	fmt.Println(multiples(1000))
}

File: p-1/solution.ts
function multiples(start: number): number {
  let sum = 0; 
  for (let i = start - 1; i > 0; i--) {
    if(i % 3 === 0 || i % 5 === 0) {
      sum += i
    }
  }

  return sum; 
}; 

console.log(multiples(10), multiples(100), multiples(1000));